tests/dashbio_demos/dash-pileup/app.py

```diff
@@ -0,0 +1,200 @@
+            {
+                'viz':'pileup',
+                'label': 'Luminal Mouse cells',
+                'source': 'bam',
+                'sourceOptions': luminal_bam,
+            },
+        ]
+}
+
+HOSTED_CASE_DICT = {
+    'rna-differential': rna_differential,
+}
+
+HOSTED_USE_CASES = [
+    {'value': 'rna-differential', 'label': 'Differential RNA-seq'},
+]
+
+def layout():
+    return html.Div(id='pileup-body', className='app-body', children=[
+        html.Div(id='pileup-control-tabs', className='control-tabs', children=[
+            dcc.Tabs(
+                id='pileup-tabs',
+                value='data',
+                children=[
+                    dcc.Tab(
+                        label='Data',
+                        value='data',
+                        children=html.Div(className='control-tab', children=[
+                            dcc.Graph(
+                                id='pileup-dashbio-volcanoplot',
+                                figure=dash_bio.VolcanoPlot(
+                                    dataframe=DE_dataframe,
+                                            effect_size='log2FoldChange',
+                                            title="Differentially Expressed Genes",
+                                            genomewideline_value=-np.log10(0.05),
+                                            p='padj',
+                                            snp='SNP',
+                                            gene='Gene',
+                                )
+                            )
+                        ])
+                    ),
+                    dcc.Tab(
+                        label='About',
+                        value='what-is',
+                        children=html.Div(className='control-tab', children=[
+                            html.H4(className='what-is', children='What is pileup.js?'),
+                            dcc.Markdown(
+                                """
+                                The Dash pileup.js component is a high-performance genomics
+                                data visualization component developed originally by the Hammer Lab
+                                (https://github.com/hammerlab/pileup.js). pileup.js
+                                supports visualization of genomic file formats, such as vcfs,
+                                bam, and bigbed files. pileup.js additionally allows flexible interaction
+                                with non-standard data formats. Users can visualize GA4GH JSON formatted
+                                alignments, features and variants. Users can also connect with and visualize
+                                data stored in GA4GH formatted data stores.
+                                """
+                            )
+                        ])
+                    )
+                ]
+            )
+        ]),
+        dcc.Loading(className='dashbio-loading', id='pileup-output', children =
```

Comment:
```suggestion
        dcc.Loading(parent_className='dashbio-loading', id='pileup-output', children =
```
Update for an issue we had with `dcc.Loading` css styling breaking.